api2-model/src/main/java/eu/europeana/api2/v2/model/json/view/BriefView.java

```diff
@@ -91,9 +91,9 @@ public String[] getDataProvider() {
 
     @Override
     public String[] getLanguage() {
-        if (profiles.contains(Profile.MINIMAL)) {
-            return new String[0];
-        }
+//        if (profiles.contains(Profile.MINIMAL)) {
+//            return new String[0];
+//        }
```

Comment:
In a few months time, we probably don't know why we commented this out, so either remove permanently or comment why this was temporarily commented out.